site/content/docs/user/loadbalancer.md

```diff
@@ -26,16 +26,10 @@ description: |-
 
 ## Installing metallb using default manifests
 
-### Create the metallb namespace
```

Comment:
Should we add a small note, explaining this history that "since version x metallb moved to CRDs" ...

Reply:
will do!